packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -320,10 +331,318 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
+   * @param {IFieldResolver} queryObj - Query resolver object
+   */
+  protected addQueryResolvers(model: ModelDefinition, queryObj: any) {
+    const modelType = model.graphqlType;
+
+    if (model.crudOptions.findOne) {
+      this.addFindOneQueryResolver(modelType, queryObj)
+    }
+    if (model.crudOptions.find) {
+      this.addFindQueryResolver(modelType, queryObj)
+    }
+  }
+
+  /**
+   * Create Mutation resolver fields
+   *
+   * @param {ModelDefinition} model - The model definition with CRUD config and GraphQL typr
+   * @param {IFieldResolver} mutationObj - Mutation resolver object
+   */
+  protected addMutationResolvers(model: ModelDefinition, mutationObj: any) {
+    const modelType = model.graphqlType;
+
+    if (model.crudOptions.create) {
+      this.addCreateMutationResolver(modelType, mutationObj)
+    }
+    if (model.crudOptions.update) {
+      this.addUpdateMutationResolver(modelType, mutationObj)
+    }
+    if (model.crudOptions.delete) {
+      this.addDeleteMutationResolver(modelType, mutationObj)
+    }
+  }
+
+  /**
+   * Create Subscription resolver fields
+   *
+   * @param {ModelDefinition} model - The model definition with CRUD config and GraphQL typr
+   * @param {IFieldResolver} subscriptionObj - Subscription resolver object
+   */
+  protected addSubscriptionResolvers(model: ModelDefinition, subscriptionObj: any) {
+    const modelType = model.graphqlType;
+
+    if (model.crudOptions.create && model.crudOptions.subCreate) {
+      this.addCreateSubscriptionResolver(modelType, subscriptionObj)
+    }
+    if (model.crudOptions.update && model.crudOptions.subUpdate) {
+      this.addUpdateSubscriptionResolver(modelType, subscriptionObj)
+    }
+    if (model.crudOptions.delete && model.crudOptions.subDelete) {
+      this.addDeleteSubscriptionResolver(modelType, subscriptionObj)
+    }
+  }
+
+  /**
+   * Create relationship resolver fields
+   *
+   * @param {FieldRelationshipMetadata[]} relationships - Model relationship metadata
+   * @param {IResolvers} resolverObj - Resolvers object
+   */
+  protected addRelationshipResolvers(relationships: FieldRelationshipMetadata[], resolverObj: IResolvers) {
+    if (!relationships.length) { return undefined }
+
+    for (const relationship of relationships) {
+      if (relationship.kind === 'oneToMany') {
+        this.addOneToManyResolver(relationship, resolverObj)
+      } else {
+        this.addOneToOneResolver(relationship, resolverObj)
+      }
+    }
+  }
+
+  /**
+   * Creates a Create mutation resolver field
+   *
+   * @param {GraphQLObjectType} modelType - Model GraphQL object type
+   * @param {IFieldResolver} mutationObj - Mutation resolver object
+   */
+  protected addCreateMutationResolver(modelType: GraphQLObjectType, mutationObj: any) {
+    const modelName = modelType.name;
+    const resolverCreateField = getFieldName(modelName, GraphbackOperationType.CREATE);
+
+    mutationObj[resolverCreateField] = (_: any, args: any, context: any) => {
```

Comment:
I think the context should be properly typed to avoid problems like the one solved by this https://github.com/aerogear/graphback/pull/1408#discussion_r437495973

```suggestion
    mutationObj[resolverCreateField] = (_: any, args: any, context: any) => {
```